os_is_not_exist_matcher.go

```diff
@@ -0,0 +1,32 @@
+package gomegamatchers
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/onsi/gomega/types"
+)
+
+func BeAnOsIsNotExistError() types.GomegaMatcher {
+	return &fileIsNotFoundMatcher{}
+}
+
+type fileIsNotFoundMatcher struct {
```

Comment:
Why is it called `fileIsNotFoundMatcher` if the matcher, shouldn't it be `osIsNotExistErrorMatcher` or something?